src/main/webapp/i18n/en/editor.json

```diff
@@ -17,7 +17,7 @@
             "unsubmittedChanges": "There are unsubmitted changes.",
             "submittingChanges": "Submitting changes.",
             "preparingForSubmit": "Preparing for submit.",
-            "submit": "Submit",
+            "submit": "Compile and run",
```

Comment:
please revert this change and rather provide it as tooltip

Reply:
Reverted